Sprite Changes

CharlieboyX opened the pull request:
sprite changes and .ips path changes

CharlieboyX committed:
Sprite Changes

sprite changes and path changes

CharlieboyX committed:
Sprite Changes

sprite changes and path changes

dagit commented:
I was helping Charlie start this PR. It's still a bit of a work in progress but I wanted to make sure @squid-man you could see what he's working on.

CharlieboyX committed:
start of the boss swaps library

A few boss swaps, more to come

CharlieboyX committed:
start of the boss swaps library

A few boss swaps, more to come

CharlieboyX committed:
Update CutmanBSD_Flashman.ips

updated CutmanBSD_Flashman.ips walk cycle animation to "working"

CharlieboyX committed:
Update CutmanBSD_Flashman.ips

updated CutmanBSD_Flashman.ips walk cycle animation to "working"

CharlieboyX committed:
Added RedSuperArros_Item2.ips

added a .ips to swap i2 with Super Arrow from MM5. This is a graphic swap, did not swap the colors(yet).

CharlieboyX committed:
Added RedSuperArros_Item2.ips

added a .ips to swap i2 with Super Arrow from MM5. This is a graphic swap, did not swap the colors(yet).

CharlieboyX committed:
Update CutmanBSD_Flashman.ips

More CutmanBSD_Flashman.ips cleanup.

CharlieboyX committed:
Update CutmanBSD_Flashman.ips

More CutmanBSD_Flashman.ips cleanup.

CharlieboyX committed:
Added More CutmanBSD boss swap .ips

CharlieboyX committed:
Added More CutmanBSD boss swap .ips

CharlieboyX committed:
added Peercast_MetalMan SuperArrow_I2 and Rollchan2_I123

CharlieboyX committed:
added Peercast_MetalMan SuperArrow_I2 and Rollchan2_I123

CharlieboyX committed:
added megakyle's version of luckyman 2

CharlieboyX committed:
added megakyle's version of luckyman 2

CharlieboyX committed:
Updated megakyle's luckyman.ips

CharlieboyX committed:
Updated megakyle's luckyman.ips

CharlieboyX committed:
Added Peercast robos, DrLightWilyReplacement, more I1,I2,I3, MM2 RemixMegaman, MM2RemixLifebar

CharlieboyX committed:
Added Peercast robos, DrLightWilyReplacement, more I1,I2,I3, MM2 RemixMegaman, MM2RemixLifebar

CharlieboyX committed:
Added i1,i2,i3 from Peercast/MM2Remix (missing from previous update)

CharlieboyX committed:
Added i1,i2,i3 from Peercast/MM2Remix (missing from previous update)

CharlieboyX committed:
Merge pull request #1 from megakyle83/develop

Charliboy! Lets share

CharlieboyX committed:
added README.txt to all folders

CharlieboyX committed:
added README.txt to all folders

CharlieboyX committed:
Merge branch 'charlieboy_MM2spriteproject' into develop

Commit:
Updating to the latest Avalonia release.

Commit:
Resizing the main window to fit with the fluent theme.

Commit:
Resizing the main window.

CharlieboyX committed:
Merge branch 'develop' of https://github.com/CharlieboyX/MegaMan2Randomizer2 into develop

CharlieboyX committed:
Added Prodigy Sprites Hatsune_Miku

CharlieboyX committed:
Added Grayzone Robot Masters, Added Prodigy Sprite swaps on some enemies and Added CB Metroid_M445 replacement

CharlieboyX committed:
Added Breakman, 1up met, and a Prodigy PiPi Bomb

CharlieboyX committed:
More Prodigy sprite swaps added

CharlieboyX committed:
Added FinalFantasyFighter,ArcadeMario, Bulletbill Item2, Exaust Alien, teleport cleanup, and repathed Wily&Alien folder to have Wily & Alien seperate, or both

CharlieboyX committed:
More sprites added! Some CB and some Prodigy

CharlieboyX committed:
Megakyle's MM2quint.ips and more Prodigy enemy sprites!

CharlieboyX committed:
Added README.txt giving credit to LVCreed for MegaRan character sprite added with previous commit

CharlieboyX committed:
Added GodWorld Weapons, more enemy sprites and some fixes

CharlieboyX committed:
Added Samus, Luigi, and another Miku with hud elements

CharlieboyX committed:
Added No constancy Weapons and a few others, also some enemies

CharlieboyX committed:
added Megakyles cutmanBSD cutscenes

squid-man committed:
Merge branch 'develop' into charlieboy_MM2spriteproject

Commit:
Updating the resource locations for the original character sprites.
